web-api/src/business/useCaseHelper/docketEntry/fileAndServeDocumentOnOneCase.ts

```diff
@@ -87,11 +88,18 @@ export const fileAndServeDocumentOnOneCase = async ({
   });
 };
 
-const completeWorkItem = async ({ user, workItemToUpdate }) => {
+const completeWorkItem = async ({
+  user,
+  workItemToUpdate,
+  docketEntryEntity,
+}) => {
   workItemToUpdate.assignToUser({
     assigneeId: user.userId,
     assigneeName: user.name,
-    section: user.section,
+    section: WorkItem.getWorkItemSectionFromUserSection({
+      section: user.section,
+      documentTitle: docketEntryEntity.documentTitle,
+    }),
```

Comment:
The basic idea: anytime section depends on user.section, we need to call this function. I _could_ put this logic directly in `assignToUser` and/or in the constructor, but it felt messy passing in a `docketEntry` field like that. I would rather put the onus on the person constructing/calling the method and use better typing to enforce which sections are allowed.